src/ProjectSystemTools/BrokeredService.pkgdef

```diff
@@ -0,0 +1,3 @@
+[$RootKey$\BrokeredServices\LoggerService\1.0]
+@="{6BEB9F10-4E64-41B7-B4E3-1BE77CD63DD4}"
+"audience"=dword:00000503
```

Comment:
What is this pkgdef doing?

Reply:
This is needed for the frontend to ask for the Codespaces service from project-system